packages/share/ios/Classes/SharePlugin.m

```diff
@@ -6,6 +6,49 @@
 
 static NSString *const PLATFORM_CHANNEL = @"plugins.flutter.io/share";
 
+@interface ShareData : NSObject <UIActivityItemSource>
+
+@property(readonly, nonatomic, copy) NSString *subject;
+@property(readonly, nonatomic, copy) NSString *text;
+
+- (instancetype)initWithSubject:(NSString *)subject text:(NSString *)text NS_DESIGNATED_INITIALIZER;
+
+- (instancetype)init __attribute__((unavailable("Use initWithSubject:text: instead")));
+
+@end
+
+@implementation ShareData
+
+- (instancetype)init {
+  [super doesNotRecognizeSelector:_cmd];
+  return nil;
+}
+
+- (instancetype)initWithSubject:(NSString *)subject text:(NSString *)text {
+  self = [super init];
+  if (self) {
+    _subject = subject;
+    _text = text;
+  }
+  return self;
+}
+
+- (id)activityViewControllerPlaceholderItem:(UIActivityViewController *)activityViewController {
+  return @"";
```

Comment:
curious - should this be null?

Reply:
Per https://developer.apple.com/documentation/uikit/uiactivityitemsource/1620458-activityviewcontrollerplaceholde?language=objc

` It should be one that the activity can handle otherwise you may get an activity with empty content. `

So setting it to null/nil will result the activityController showing an empty content (manually tested)